spec/features/users/google_oauth_spec.rb

```diff
@@ -48,6 +48,7 @@
       visit '/login'
       click_on 'Sign in with Google'
       expect(current_path).to eq('/')
+      expect(User.last.calendar_id.empty?).to eq(false) 
```

Comment:
Add test that signing in through `googleoauth` now adds the `calendar_id` to the `User`